### PR TITLE
Search analytics improvements

### PR DESCRIPTION
**Pull request by dojutsu-user**
This is a follow up PR from #6019

### CODE REVIEW

**Comment by dojutsu-user** [comment]
@ericholscher 
I have made the suggested changes.

> Not sure if we really even want to cascade these deletes. Is there a reason we don't want to store Version here as a string, so we can keep them forever even if a version is deleted? ([comment](https://github.com/readthedocs/readthedocs.org/pull/6019#discussion_r311712452))

We can definitely do these. But we are keeping nothing for `forever` -- we delete the old queries every day. So I think that won't be of much benefit?

**Comment by dojutsu-user** [comment]
We do have some queries to optimize if they break in the prod. :crossed_fingers:

File: readthedocs/projects/views/base.py [hunk 1]
```diff
@@ -115,3 +115,17 @@ def post(self, request, *args, **kwargs):
...
+    """
+    A class that implements just the write method of the file-like interface.
+
+    This class can be used for generating StreamingHttpResponse.
+    See: https://docs.djangoproject.com/en/2.2/howto/outputting-csv/#streaming-large-csv-files
+    """
+
+    def write(self, value):
+        """Write the value by returning it, instead of storing in a buffer."""
+        return value
```
Comment (ericholscher) [inline readthedocs/projects/views/base.py]: I'd probably put this in a `utils.py` class, not a base View.

File: readthedocs/templates/projects/projects_search_analytics.html [hunk 2]
```diff
@@ -13,13 +13,16 @@
 {% block project_edit_content %}
   {% if show_analytics %}
 
-    <h3>{% trans "Top Queries:" %}</h3>
+    <h3>{% trans "Top queries of all time" %}</h3>
```
Comment (ericholscher) [inline readthedocs/templates/projects/projects_search_analytics.html]: I don't "all time" is correct. I think the previous text is fine.

File: readthedocs/projects/views/base.py (hunk 1 above)
Comment (dojutsu-user) [reply readthedocs/projects/views/base.py]: Done. :+1:

**Comment by ericholscher** [comment]
Looks good, will ship this out today. 👍

**Pull request merged**